### DISCUSSION TITLE
Legal to use as a company

### DISCUSSION THREAD

**Discussion by Nollknolle** [dani-garcia/vaultwarden#2771] [Q&A] [answered]
Hello,
Is it legal to use this Server as a small company?

**Comment by RealOrangeOne** [comment, marked as answer]
So long as you follow the license, sure. What are you concerned about?

**Reply by Nollknolle** [reply]
I am afraid that I can get legal problems if I use vaultwarden in my company.

**Reply by RealOrangeOne** [reply]
Not quite an answer. What problems do you foresee? What do you imagine could be an issue for your company / use of vaultwarden?

**Reply by Nollknolle** [reply]
I have concerns because some features require a license and are unlocked through Vaultwarden.

**Reply by RealOrangeOne** [reply]
They require a license for bitwarden itself, vaultwarden is a re-implementation, so it's ok. If it was ok, vaultwarden wouldn't really exist, and it'd be just as illegal for users to use it than companies.

**Reply by Nollknolle** [reply]
Ok, then I am reassured. Thank you very much!

**Comment by bokkabonga** [comment]
As far as i know it´s completely fine and legal to use vaultwarden as a company. However there are some features that are not implemented, that (especially with bigger companies) you might see as necessary.

**Reply by Nollknolle** [reply]
Thank you for your answer!
